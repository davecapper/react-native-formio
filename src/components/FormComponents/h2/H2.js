import React from 'react';
import {Text} from '@rneui/themed';
import PropTypes from 'prop-types';
import styles from './styles';

const H2 = (props) => (
  <Text style={styles.header2}>
    {props.children}
  </Text>
);

H2.propTypes = {
  children: PropTypes.node
};

export default H2;
