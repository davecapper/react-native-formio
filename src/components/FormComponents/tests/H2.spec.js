import React from 'react';
import renderer from 'react-test-renderer';
import H2 from '../h2/H2';
import {Text} from '@rneui/themed';

describe('H#', () => {
  describe(' H2 component', () => {
    it('Renders a basic H2 component', () => {
      const element = renderer.create(<H2><Text>Content</Text></H2>);

      expect(element).toMatchSnapshot();
    });
  });
});
