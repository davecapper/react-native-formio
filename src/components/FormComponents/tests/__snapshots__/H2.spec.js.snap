// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H#  H2 component Renders a basic H2 component 1`] = `
<Text
  accessibilityRole="text"
  style={
    {
      "color": "#242424",
      "fontSize": 28,
      "letterSpacing": 0.36,
      "lineHeight": 34,
      "margin": 10,
    }
  }
>
  <Text
    accessibilityRole="text"
    style={
      {
        "color": "#242424",
      }
    }
  >
    Content
  </Text>
</Text>
`;
